### PR TITLE
Add ocular

### PR DESCRIPTION
**Pull request by simonwep**
Regarding "Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects." - the app _uses_ Go server side and JS in the frontend but can be _deployed_ by using only docker... should I remove all other platforms besides docker?

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [ ] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by kokomo123** [comment]
If its the only installation method with docker, then yes. If you could compile / install using go or javascript, then yeah you can keep those in, hope that helps!

File: software/ocular.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: "Ocular"
+website_url: "https://github.com/simonwep/ocular"
+source_code_url: "https://github.com/simonwep/ocular"
+description: "Simplistic and straight-forward budgeting app to track your budget across months and years."
+licenses:
+  - MIT
+platforms:
+  - Javascript
+  - Go
```
Comment (kokomo123) [inline software/ocular.yml:9]:
```suggestion

```
Considering only installation method is docker, this would be good.

Comment (simonwep) [reply software/ocular.yml]: Sure thing, thank you for clarifying! :)

**Review by kokomo123 — APPROVED** [review]
Thanks for correcting, LGTM, thanks for your contribution as well!

**Review by Rabenherz112 — APPROVED** [review]
LGTM

**Review by nodiscc — APPROVED** [review]

**Review by ryangurn — APPROVED** [review]
LGTM

**Pull request merged**